src/app/ui/player-quest/player-quest.service.ts

```diff
@@ -270,12 +273,19 @@ export class PlayerQuestService {
         eligibleForGoodWorkBadge = (counter.points >= 50);
         eligibleForOutstandingWorkBadge = counter.quests > 4;
 
+        // if the month is April (end of a season), check for you're a star eligibility
+        if (monthName.toLowerCase() === 'apr' && !youreAStar) {
+          eligibleForYoureStarBadge = this.badgeService.isEligibleForYoureStar(monthlyCounter, youreAStar);
+          youreAStar = youreAStar || eligibleForYoureStarBadge; // make sure that if youre a star is already given, retain the marker.
```

Comment:
no longer need to pass youreAStar + you can directly assign 
```suggestion
          youreAStar = eligibleForYoureStarBadge = this.badgeService.isEligibleForYoureStar(monthlyCounter);
```